Gemfile.lock

```diff
@@ -0,0 +1,313 @@
+GEM
```

Comment:
Please remove

Reply:
It's autogenerated, so will modify Gemfile and bundle and push changes.